<launch>
               
		<node name="flight_controller" pkg="tiltrotor_drone" type="flight_control_exe" output="screen">
		<param name="node_rate" value="50.0"/>
		<param name="drone_name" value="iris_tiltrotor"/>
		<param name="kp_x" value="1"/>
		<param name="ki_x" value="0"/>
		<param name="kd_x" value="0"/>
		<param name="kp_y" value="1.5"/>
		<param name="ki_y" value="0"/>
		<param name="kd_y" value="0.3"/>
		<param name="kp_z" value="1"/>
		<param name="ki_z" value="0"/>
		<param name="kd_z" value="0"/>
		<param name="kp_vy" value="10"/>
		<param name="ki_vy" value="0"/>
		<param name="kd_vy" value="0"/>
		<param name="kp_roll" value="4"/>
		<param name="ki_roll" value="0"/>
		<param name="kd_roll" value="0.3"/>
		<param name="kp_yaw" value="0.5"/>
		<param name="ki_yaw" value="0.0"/>
		<param name="kd_yaw" value="0"/>
		<param name="lim_ux" value="20.0"/>
		<param name="lim_uy" value="20.0"/>
		<param name="lim_uz" value="10.0"/>
		<param name="kp_tau" value="0.3"/>
		<param name="ki_tau" value="0.0"/>
		<param name="kd_tau" value="0"/>
		<param name="m" value="5.1664"/>
		<param name="I" value="0.15"/>
		<param name="g" value="9.8066"/>
	</node>
</launch>
